src/main/scala/cromwell/binding/expression/WdlFunctions.scala

```diff
@@ -16,16 +16,14 @@ trait WdlFunctions[T] {
   }
 
   /**
-    * Given a path to a file, return the contents
-    * of that file.
-    * @param path - path to the file
-    * @return - Contents of the file
-    * @throws UnsupportedOperationException if the WDL value can
-    *         not be interpreted as a file
-    * @throws NotImplementedError if the backend did not implement
-    *         this method
-    */
-  def fileContentsToString(path: String): String = throw new NotImplementedError("fileContentsToString() is unimplemented")
+    * Path where to write files created by engine functions (write_*).
+   */
+  def tempFilePath: String = throw new NotImplementedError("tempFilePath() is unimplemented")
```

Comment:
But when I read it, ??? looks like a temporary placeholder whereas this looks like a conscious decision.